spec/api/comment_spec.rb

```diff
@@ -7,38 +7,48 @@
 
   describe "comments" do
     before(:each) { init_without_subscriptions }
+
     describe "GET /api/v1/comments/:comment_id" do
       it "returns JSON" do
-        comment = Comment.first
-        get "/api/v1/comments/#{comment.id}"
-        last_response.should be_ok
-        last_response.content_type.should == "application/json;charset=utf-8"
+        Comment.each do |comment|
```

Comment:
Why the change to validating each comment in the database vs only testing on a single comment?